apps/src/templates/projects/ProjectWidgetWithData.jsx

```diff
@@ -30,11 +31,14 @@ const ProjectWidgetWithData = React.createClass({
   },
 
   render() {
+    const { isRtl } = this.props;
+
```

Comment:
nit: you're only using this in one place, it doesn't feel necessary to make it a constant.
Count just say `isRtl={this.props.isRtl}`